### DISCUSSION TITLE
KeyError:'SU2_RUN' on PATHs for versions 6.2.0 and 5.0.0

### DISCUSSION THREAD

**Discussion by aviony** [su2code/SU2#1275] [Q&A]
Hi there, 

I am trying to get SU2 work with different software, but the path added to the bashrc does not make it recognized in the terminal of the particular software (Engineering Sketch Pad) I try to run, but I when run echo $PATH on other terminals, the PATH is indeed present.  I have already tried what it says on the main website of SU2 as well. 

![Screenshot_20210430_114156](https://user-images.githubusercontent.com/80710208/116678111-44933f80-a9a9-11eb-90ff-f71f993ae147.png)

![path_bashrc](https://user-images.githubusercontent.com/80710208/116678350-7efcdc80-a9a9-11eb-85b7-401d6d6760d2.png)

Can you please help me with this issue?

Thank you beforehand!

**Comment by TobiKattmann** [comment]
Can you please try to `export SU2_RUN="<your path>"` in your shell instance directly before calling  your script which fails. The $PATH is not really the issue here, the evironmental variable SU2_RUN is missing. So checking `echo $SU2_RUN` is what needs to work, not the $PATH. But the .bashrc looks good to me. Let me know if that helped somehow.
